{{ cookiecutter.project_slug }}/.pre-commit-config.yaml

```diff
@@ -41,6 +41,8 @@ repos:
     hooks:
     - id: mypy
       exclude: ^tests/.*$
+      additional_dependencies:
```

Comment:
as we discussed this logic is a bit confusing to me, but I'm approving as it not impacted by the PR